components/textbook-demo/TextbookDemoContentMenuSection.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <section class="content-menu-section">
+    <div class="bx--grid">
+      <AppMegaDropdownMenu :content="dropdownMenuContent" />
+    </div>
+  </section>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { Component } from 'vue-property-decorator'
+import { TEXTBOOK_DEMO_MEGA_MENU } from '~/constants/megaMenuLinks'
+
+@Component
+export default class TextbookDemoContentMenuSection extends Vue {
```

Comment:
This is no longer a demo, so we should stop naming things with "textbook-demo" if possible. Maybe we can name this component `AppMegaDropdownMenuNavbar` to show the connection with the `AppMegaDropdownMenu`

Reply:
As discussed offline, we'll be renaming the instances of "textbook demo" to "textbook" with https://github.com/Qiskit/qiskit.org/issues/1871.

Reply:
As discussed offline, I agree that the connection to the `AppMegaDropdownMenu` component is important to describe this component, but I'd argue that the fact that this component displays the content specific to the textbook is as important if not even more important.

If a _perfect_ world (one where the length of the component's name is irrelevant) I'd probably suggest something like `TextbookContentMegaDropdownMenuSection` (<configuration><element wrapper ><kind of component>).